lib/manifest.ps1

```diff
@@ -14,14 +14,15 @@ function parse_json($path) {
 function url_manifest($url) {
     $str = $null
     try {
+        if (!(Get-Command Get-UserAgent -ErrorAction SilentlyContinue)) {
```

Comment:
Note: this `if` statement imports Get-UserAgent if a \*suspected\* race condition causes the function to be called before it's imported by a separate script. I don't know why so many of Scoop's scripts call functions without explicitly importing them beforehand. That's a refactor for another day.